group_vars/location_moll/networks.yml

```diff
@@ -0,0 +1,59 @@
+---
+ipv6_prefix: "2001:bf7:750:5200::/56"
+
+# B0russia got following prefixes:
+# Router: 10.31.162.96/27
+# --MGMT: 10.31.162.96/28
+# --MESH: 10.31.162.112/28
+# --DHCP: 10.31.163.0/24
+
+networks:
+  - vid: 40
+    role: dhcp
+    inbound_filtering: true
+    enforce_client_isolation: true
+    prefix: 10.31.163.0/24
```

Comment:
Is it really really necessary to have a /24? That means around 250 adresses. We don't have that much IPv4-adresses left. Is a /25 enough maybe?

Reply:
It was said that 100 people are expected to join the center. That is why I would like to use /24. However, it is also just temporary and is deleted afterwards again.

Reply:
Sounds good.